nuxeo-features/nuxeo-elasticsearch/nuxeo-elasticsearch-audit/src/test/resources/es-audit-pageprovider-test-contrib.xml

```diff
@@ -3,97 +3,131 @@
 <component name="org.nuxeo.elasticsearch.PageProviderservice.test.contrib">
 
   <extension target="org.nuxeo.ecm.platform.query.api.PageProviderService"
-    point="providers">
+             point="providers">
 
     <genericPageProvider name="SimpleESAuditPP"
-      class="org.nuxeo.elasticsearch.audit.pageprovider.ESAuditPageProvider">
-       <pattern>
-       {
-				 "query": {
-           "bool": {
-             "must": {
-               "match": {
-                  "docPath": {
-                    "query": "/dummy"
+                         class="org.nuxeo.elasticsearch.audit.pageprovider.ESAuditPageProvider">
+      <pattern>
+        {
+          "query":{
```

Comment:
Why are you removing spaces after `:`?

Reply:
i used a `json` plugin formater and i didn't pay attention that it removes the space